pkg/render/node.go

```diff
@@ -653,6 +656,7 @@ func (c *nodeComponent) cniContainer() v1.Container {
 	cniVolumeMounts := []v1.VolumeMount{
 		{MountPath: "/host/opt/cni/bin", Name: "cni-bin-dir"},
 		{MountPath: "/host/etc/cni/net.d", Name: "cni-net-dir"},
+		{MountPath: "/host/var/log/calico/cni", Name: "cni-log-dir"},
```

Comment:
Why does the container need this MountPath? Is there some action that the install does that needs to access the log dir?

Reply:
Yeah, this isn't needed. I mistook the installer container with where it was actually running.